src/styles/partials/_global.scss

```diff
@@ -40,3 +40,22 @@ a {
     text-decoration: none;
   }
 }
+
+input, select, textarea {
+	display: block;
+	background: #fbfbfb;
+	border: 1px solid #eee;
+	border-radius: 4px;
+	color: #2683FF;
+	padding: 12px 18px 10px;
+	margin: 25px auto;
+	width: 100%;
+	max-width: 320px;
+  font-size: 16px;
```

Comment:
the indentation here looks off.

Reply:
Hmmm... Indentation looks fine on my local file. I'll make sure to keep an eye out for this on the next commit. Good catch!